src/uu/du/src/du.rs

```diff
@@ -913,6 +913,9 @@ impl FromStr for Threshold {
         let size = parse_size(&s[offset..])?;
 
         if s.starts_with('-') {
+            if size == 0 {
```

Comment:
Maybe you could add a comment here explaining why `-0` does not make sense. From what I can gather it's because the you would be hiding everything with a size > 0, i.e. almost everything.

Reply:
Noted. Thanks!